solidity/contracts/token/libs/ValueTransferBridge.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity >=0.8.0;
+
+struct Quote {
+    address token;
+    uint256 amount;
+}
+
+abstract contract ValueTransferBridge {
+    function quoteTransferRemote(
+        uint32 destinationDomain,
+        bytes32 recipient,
+        uint amountOut
+    ) public view virtual returns (Quote[] memory) {}
```

Comment:
still think memory arrays are pretty annoying to work with and gas inefficient
what bridge implementations have multiple fees in different denominations? I'm personally not aware of one

Reply:
Yeah, probably not too useful at the moment. I think we just want to give bridges the flexibility of charging multiple tokens (e.g. charge fees in the token being bridged).